osu.Game.Rulesets.Taiko/Skinning/Legacy/TaikoLegacySkinTransformer.cs

```diff
@@ -35,7 +35,7 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
             {
                 // if a taiko skin is providing explosion sprites, hide the judgements completely
                 if (hasExplosion.Value)
-                    return Drawable.Empty();
+                    return Drawable.Empty().With(d => d.LifetimeEnd = double.MinValue);
```

Comment:
does `.Expire()` also work here? may read better if so.

Reply:
It does, yeah. The only reason I wrote it this way is because this idiom is used in another place in the same file. I'll update both to that.